runners/reference/job-server/build.gradle

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import groovy.json.JsonOutput
+
+apply plugin: org.apache.beam.gradle.BeamModulePlugin
+applyJavaNature(
+  validateShadowJar: false,
+  shadowClosure: {
+  }
+)
+
+description = "Apache Beam :: Runners :: Reference :: Job Server"
+
+apply plugin: "application"
+
+mainClassName = "org.apache.beam.runners.direct.portable.job.ReferenceRunnerJobServer"
+
+dependencies {
+  compile project(path: ":beam-runners-direct-java", configuration: "shadow")
+  compile project(path: ":beam-runners-java-fn-execution", configuration: "shadow")
+  compile library.java.slf4j_simple
+}
+
+run {
```

Comment:
Shall we name it to `runShadow` for consistency

Reply:
I don't think it's necessary because in this case the JobServer works with just run. It feels odd using the shaded version just for consistency (unless there's some other advantage to using runShadow that I don't know).